public_html/wp-content/plugins/camptix/camptix.php

```diff
@@ -8595,6 +8597,83 @@ public function is_wordcamp_closed() {
 	public function has_tickets_available() {
 		return $this->number_available_tickets() > 0;
 	}
+
+	/**
+	 * Log the purchased ticket information on the user's profile.
+	 *
+	 * @param object $attendee The attendee object containing ticket information.
+	 */
+	function log_ticket_purchase_on_user_profile( $attendee ) {
+		$user_id = get_current_user_id();
+		$purchase_history = get_user_meta( $user_id, 'wordcamp_ticket_history', true );
```

Comment:
should this be a single user meta or can it be multiple?

Reply:
Using a single entry works for how I currently store the data. But yeah I should save it in a way that allows multiple access as it would make the structure and data access clearer.